recipes/sphinx-autobuild/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sphinx-autobuild" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "2f9262d7a35f80a18c3bcb03b2bf5a83f0a5e88b75ad922b3b1cee512c7e5cd2" %}
+
+package:
+  name: sphinx-autobuild
```

Comment:
For the `name` and `version` here, you should use the jinja template variables. See:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L13-L14